content/blog/announcing-spin-3-5.md

```diff
@@ -0,0 +1,152 @@
+title = "Announcing Spin v3.5"
+date = "2025-11-10T01:27:05.449167Z"
+template = "blog_post"
+tags = []
+
+[extra]
+type = "post"
+author = "Michelle Dhanani"
+
+---
+
+The CNCF Spin project recently released [Spin v3.5.0](https://github.com/spinframework/spin/releases/tag/v3.5.0) which includes support for the WASI Preview 3 (WASIp3) release candidate along with several developer experience enhancements! In this blog post, we’ll dive into a few exciting features of this release and highlight an example of what is enabled with WASIp3 in Spin.
```

Comment:
The "p" in "WASIp3" doesn't stand for "Preview". "Point" if one must, but really it's just "p" :)

```suggestion
The CNCF Spin project recently released [Spin v3.5.0](https://github.com/spinframework/spin/releases/tag/v3.5.0) which includes support for a release candidate of the upcoming WASIp3 release, along with several developer experience enhancements! In this blog post, we’ll dive into a few exciting features of this release and highlight an example of what is enabled with WASIp3 in Spin.
```